go.mod

```diff
@@ -13,16 +11,17 @@ require (
 	github.com/erh/viamupnp v0.0.0-20250225174543-39c68c119b3e
 	github.com/gofrs/uuid v4.2.0+incompatible
 	github.com/golangci/golangci-lint v1.61.0
+	github.com/hexbabe/sean-onvif v0.0.10-0.20250417162035-97e81b8c0ba9
```

Comment:
Should logger changes here be added to viamonvif?

Reply:
Can make a follow up ticket for that fs